core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/naming/validation/OicrLibraryAliasValidator.java

```diff
@@ -6,12 +6,19 @@
 
   // PROJ_0001_...
   private static final String commonPrefix = "([A-Z\\d]{3,5})_(\\d{3,6}|\\d[CR]\\d{1,2})_";
-  // ...Pa_P_PE_700_WG
-  private static final String illuminaRegex = "(nn|[A-Z][a-z])_([A-Zn])_(SE|PE|MP|TR|\\?\\?)_(nn|\\d{2,6}|\\dK)_(TS|EX|CH|BS|WG|TR|WT|SM|MR|AS|\\?\\?)";
+  // ...Pa_P...
+  private static final String tissueMid = "(nn|[A-Z][a-z])_([A-Zn])_";
+
+  private static final String designCode = "(TS|EX|CH|BS|WG|TR|WT|SM|MR|AS|\\\\?\\\\?)";
+  // ...PE_700_WG
+  private static final String illuminaRegex = "(SE|PE|MP|TR|\\?\\?)_(nn|\\d{2,6}|\\dK)_" + designCode;
+  // ...1D2_WG_1
+  private static final String oxfordNanoporeRegex = "1D2?_" + designCode + "_\\d+";
   // ...20170913_1
   private static final String pacBioRegex = "\\d{8}_\\d+";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)